src/vivarium_csu_hypertension_sdc/verification_and_validation/simulant_trajectory_visualizer.py

```diff
@@ -0,0 +1,310 @@
+import math
+import matplotlib.pyplot as plt
+from matplotlib import colors
+from ipywidgets import interact, Text
+from pathlib import Path
+import pandas as pd, numpy as np
+import yaml
+from pandas.plotting import register_matplotlib_converters
+
+from vivarium_csu_hypertension_sdc.components.globals import (HYPERTENSIVE_CONTROLLED_THRESHOLD, HYPERTENSION_DRUGS,
+                                                              DOSAGE_COLUMNS, SINGLE_PILL_COLUMNS)
+
```

Comment:
I'll assume this makes pretty pictures.  We'll use expert eyeballs to debug plotting code.